src/main/java/uk/gov/hmcts/reform/pip/rules/repository/InMemoryRepository.java

```diff
@@ -20,6 +36,25 @@
 
     private static final Integer AVAILABLE_PUBLICATION = 1;
 
+
+
+    public InMemoryRepository() throws IOException {
```

Comment:
youre missing return type

Reply:
sorry not sure what return type I am missing